plugins/woocommerce/client/blocks/tests/e2e/tests/single-product-details/single-product-details.block_theme.spec.ts

```diff
@@ -68,11 +68,38 @@ test.describe( `${ blockData.slug } Block`, () => {
 		page,
 		frontendUtils,
 	} ) => {
-		const template = await requestUtils.createTemplate( 'wp_template', {
-			slug: 'single-product-v-neck-t-shirt',
-			title: 'Product Details Test',
-			content: '',
-		} );
+		let template;
+		const maxRetries = 3;
```

Comment:
Personally I don't like try catch in test like this, but it seems to be the only way to handle network issues. Should we turn it into an utility? I don't want to do that as I don't want to promote this practice.